src/Widgets/HeadlineWidget/HeadlineWidgetEditingConfig.js

```diff
@@ -60,9 +62,20 @@ Scrivito.provideEditingConfig("HeadlineWidget", {
   properties: ["style", "level", "alignment", "showDividingLine", "showMargin"],
   initialContent: {
     alignment: "left",
-    headline: "Lorem Ipsum",
+    headline: initialHeadline,
```

Comment:
Let's keep the original:

```suggestion
    headline: "Lorem Ipsum",
```